src/api/src/v1/region.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use common_base::AffectedRows;
+pub use greptime_proto::v1::region::*;
+
+/// This result struct is derived from [RegionResponse]
+#[derive(Debug)]
+pub struct RegionHandleResponse {
```

Comment:
Why not simply use the name `RegionResponse`?

Reply:
It's the name from the `proto` so the FQN `api::v1::region:: RegionResponse ` would conflict.

Naming is hard. I'm quite open to listen other options (like move this struct to another mod and name it `RegionResponse`. But where?)

Reply:
How about put it in mod `api::region`? No "`v1`", no naming conflicts. Besides, if we have "`v2`" in the future, there's no need to change all the "use" sites in the codes.

Reply:
SGTM. Let me make the change.

Reply:
Updated.